tox.ini

```diff
@@ -90,7 +90,6 @@ basepython = python3.7
 whitelist_externals = bash
 deps =
     urllib3<2.0
-    docker-compose=={[tox]docker_compose_version}
```

Comment:
we'll need to figure out what to do internally as well: we'll need the docker-compose deb installed on jenkins + devboxes before we merge this

Reply:
how can I achieve the first option? I need to deploy these changes AFAP since is blocking me from upgrading to k8s 1.22